tests/pagination/tests.py

```diff
@@ -318,12 +319,16 @@ def test_page_getitem(self):
         self.assertIsInstance(p.object_list, list)
 
     def test_paginating_unordered_queryset_raises_warning(self):
-        msg = (
+        with warnings.catch_warnings(record=True) as warns:
+            warnings.filterwarnings('always', category=UnorderedObjectListWarning)
```

Comment:
`# Prevent the RuntimeWarning subclass from appearing as an exception due to the warnings.simplefilter() in runtests.py.`